araproc/analysis/rpr.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+from scipy.ndimage import uniform_filter1d
+from araproc.framework import waveform_utilities as wfu
+
+
+def get_max_info(channel_wf, channel_time, use_debug=False):
```

Comment:
Do we need the `use_debug` flag? Maybe it's better to simplify and remove the debug statements throughout. If we need to debug we can always add print statements temporarily as needed.